src/tyr/serializers.cc

```diff
@@ -85,6 +85,20 @@ waypoint(const valhalla::Location& location, bool is_tracepoint, bool is_optimiz
   return waypoint;
 }
 
+valhalla::baldr::json::MapPtr waypoint(const valhalla::Location& location,
+                                       const valhalla::TripLeg_Edge& edge,
+                                       bool is_tracepoint,
+                                       bool is_optimized) {
+  valhalla::baldr::json::MapPtr wpt = waypoint(location, is_tracepoint, is_optimized);
+
+  // When we have access to the triplegedge we add more metadata to the waypoint.
+  std::string road_class = to_string(static_cast<valhalla::baldr::RoadClass>(edge.road_class()));
+  if (road_class != "null") {
```

Comment:
why would the road class be `null`? that shouldnt be possible! im guessing the `to_string` function just has a catch all for a number that is out of range?

Reply:
That's right the to_string function has a catch all, and based on [this lua](https://github.com/valhalla/valhalla/blob/master/lua/graph.lua#L1284-L1294) it shouldn't be possible to trigger the `"null"` case. But I figured boots and suspenders wouldn't hurt.